frontend/src/lib/components/SchemaEditor.svelte

```diff
@@ -102,7 +103,10 @@
 				schema.required.includes(argName)
 			)
 			oldArgName = argName
-			schemaModal.openDrawer()
+
+			setTimeout(() => {
```

Comment:
@rubenfiszel It's one of thoses fixes,wdyt ?

Reply:
wait, why would the setTimeout be on the openDrawer ?